### DISCUSSION TITLE
Gmail SMTP Relay error

### DISCUSSION THREAD

**Discussion by FonduemangVI** [sct/overseerr#1889] [Help] [answered]
Description

When I am trying to use the gmail smtp relay I am getting an error sending emails

Version

Running version 1.25.0

Expected Behavior

Send email notifications by gmails smtp relay

Device

Browser Microsoft Edge

The error log I am getting is this

2021-07-13T10:41:36.005Z [error][Notifications]: Error sending email notification {"recipient":"FonduemangVI","type":"TEST_NOTIFICATION","subject":"Test Notification","errorMessage":"Server terminates connection. response=421 4.7.0 Try again later, closing connection. (EHLO) o24sm3175645oou.1 - gsmtp: 421 4.7.0 Try again later, closing connection. (EHLO) o24sm3175645oou.1 - gsmtp"}

This may be similar to an error I was getting with Authelia which also had an EHLO error which was resolved by adding an HELO/EHLO Identifier. Some SMTP Servers may reject the default of localhost

Bug ticket was auto closed.
I would imagine this is a feature request or bug but any help would be appreciated

**Comment by TheCatLady** [comment, marked as answer]
This is definitely not a feature request or bug, but a support issue (it's a problem with your settings/config).

Make sure you have an application URL configured in your Overseerr settings.  See https://docs.overseerr.dev/using-overseerr/notifications/email for more details.

**Reply by FonduemangVI** [reply]
My apologies.

I have read through what you sent and updated my application URL. Emails via Gmail's SMTP are now working. Thank you for the prompt reply
